pkg/controller/util/deletionhelper/deletion_helper.go

```diff
@@ -27,6 +27,7 @@ import (
 	finalizersutil "github.com/kubernetes-sigs/federation-v2/pkg/controller/util/finalizers"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	pkgruntime "k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
Done.  This is legacy code sourced from fedv1, and it has bad habits like not aliasing apimachinery runtime to avoid potential conflict with golang's runtime package.